searx/searx
# [fix] migration from github.com/asciimoo/searx to github.com/searx/searx : fix URLs

## What does this PR do?

Update URL from:
* github.com/asciimoo/searx to github.com/searx/searx
* asciimoo.github.io/searx to searx.github.io/searx

## Why is this change important?

Avoid redirect.

## How to test this PR locally?

* check each replace URL is the right one
* check there is no missing URL

Note: the transiflex URL are not modified (I'm not sure how transiflex deals with project migration).

## Author's checklist



## Related issues
